x-pack/plugins/secops/public/apps/start_app.tsx

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { createHashHistory } from 'history';
+import React from 'react';
+import { ApolloProvider } from 'react-apollo';
+import { ThemeProvider } from 'styled-components';
+
+// TODO use theme provided from parentApp when kibana supports it
+import { EuiErrorBoundary } from '@elastic/eui';
+import * as euiVars from '@elastic/eui/dist/eui_theme_k6_light.json';
+import { AppFrontendLibs } from '../lib/lib';
```

Comment:
nit: I'd add a new line between these imports to separate the imports from an `yarn add` install the local imports. Looking through code, I prefer the style of keeping 1 blank line between imports that are from 3rd party packages and packages that are local.

optional, no changes requested.